core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/pushpull/LDAPMembershipPullActions.java

```diff
@@ -153,11 +154,8 @@ public void beforeUpdate(
         }
 
         groupDAO.findUMemberships(groupDAO.find(entity.getKey())).forEach(uMembership -> {
-            Set<String> memb = membershipsBefore.get(uMembership.getLeftEnd().getKey());
-            if (memb == null) {
-                memb = new HashSet<>();
-                membershipsBefore.put(uMembership.getLeftEnd().getKey(), memb);
-            }
+            Set<String> memb = membershipsBefore.computeIfAbsent(uMembership.getLeftEnd().getKey(),
```

Comment:
Not fundamental, but can you keep the same formatting as the other change below?